pages/en/blog/announcements/diving-into-the-nodejs-website-redesign.md

```diff
@@ -0,0 +1,118 @@
+
+## Redesign
+
+Also in 2023, the [OpenJS Foundation](https://openjsf.org/) began a redesign of their site. They hired [Hayden Bleasel](https://haydenbleasel.com/) and were kind enough to include nodejs.org in the scope of his contract. Hayden brought a modern design to the site and helped us think through the multi-faceted use cases we encountered. The result was a [Figma](https://www.figma.com/) document that we could use to guide our development. Included were UX flows, dark/light modes, page layouts, mobile viewport considerations, and a component breakdown.
+
+![A screenshot of the nodejs.org website redesign in Figma.](/static/images/blog/announcements/2024-nodejs-figma.png)
+
+Realizing the design as code was next, and an effort distributed across the world. A lot of emphasis was placed on sequential build-up of foundational design elements and structured component hierarchies. We built variants of components from day one and considered internationalization at the outset. This reduced rework and made any task accessible to a newcomer. We started building components in isolation via Storybook and [Chromatic](https://www.chromatic.com/)'s hosted instance. We choose to use [Tailwind CSS](https://tailwindcss.com/), but with an emphasis on design tokens and applied CSS. This common language helped newcomers orient to our approach and translate the Figma.
+
+[Orama](https://docs.oramasearch.com/) search puts all the site's content at a user's fingertips. They index our static content and provide lightning-fast results of API content, learning material, blog posts, and more. The team there directly contributed this integration and continues to help us deliver a superb experience. It's already hard to imagine _not_ having this feature.
+
+The reach that Node.js enjoys in our communities is important to us. As such, the old site was internationalized to almost 20 languages. An unfortunate combination of events led us to reset all translations, however. This was a hard decision, but the right one given the circumstances. We're working with [Crowdin](https://crowdin.com/) to re-establish our efforts. This will include a careful parsing of the new MDX-based content. We're looking forward to the continual internalization in the coming months.
+
+As we built and started previewing the site both on the new infrastructure and redesigned, having deeper insight into end-user behavior was invaluable. We've leveraged [Sentry](https://sentry.io/welcome/) to provide error reporting, monitoring, and diagnostic tools. This has been a great help in identifying issues and providing a better experience for our users.
```

Comment:
It is worth mentioning how Sentry's Session Playback feature allowed us to understand critical pain points (rage clicks, dead ends, heatmaps) of the current design and what we could do to improve

Reply:
addresssed via https://github.com/nodejs/nodejs.org/pull/6524/commits/1d02f9a5e798791981c3b62d044360d27b7e8f06